Sources/ClientRuntime/Middleware/PresignerShimHandler.swift

```diff
@@ -7,20 +7,20 @@
 
 typealias PresignerShimHandler = (SdkHttpRequestBuilder) -> Void
 
-struct PresignerShim<OperationStackOutput: HttpResponseBinding,
-                     OperationStackError: HttpResponseErrorBinding>: Middleware {
+struct PresignerShim<OperationStackOutput>: Middleware {
     public let id: String = "PresignerShim"
 
     private let handler: PresignerShimHandler
+    private let output: OperationStackOutput
 
-    init(handler: @escaping PresignerShimHandler) {
+    init(handler: @escaping PresignerShimHandler, output: OperationStackOutput) {
```

Comment:
Since the removal of the `HttpResponseBinding` protocol means that this shim can no longer create an `Output`, one is passed into this middleware at creation.